hosted_control_planes/hcp-disconnected/hcp-deploy-dc.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-deploy-dc"]
+include::_attributes/common-attributes.adoc[]
+= Introduction to {hcp} in a disconnected environment
+:context: hcp-deploy-dc
+
+toc::[]
+
+In the context of {hcp}, a disconnected environment is an {product-title} deployment that is not connected to the internet and that uses {hcp} as a base. You can deploy {hcp} in a disconnected environment on bare metal or {VirtProductName}.
+
+{hcp-capital} in disconnected environments function differently than in standalone {product-title}:
+
+* The control plane is in the management cluster. The control plane is where the pods of the hosted control plane are run and managed by the Control Plane Operator. 
+* The data plane is in the workers of the hosted cluster. The data plane is where the workloads and other pods run, all managed by the HostedClusterConfig Operator.
+
+Depending on where the pods are running, they are affected by the `ImageDigestMirrorSet` (IDMS) or `ImageContentSourcePolicy` (ICSP) that is created in the management cluster or by the `ImageContentSource` that is set in the `spec` field of the manifest for the hosted cluster. The `spec` field is translated into an IDMS object on the hosted cluster.
+
+You can deploy {hcp} in a disconnected environment on IPv4, IPv6, and dual-stack networks. IPv4 is one of the simplest network configurations to deploy {hcp} in a disconnected environment. IPv4 ranges require fewer external components than IPv6 or dual-stack setups. For {hcp} on {VirtProductName} in a disconnected environment, use either an IPv4 or a dual-stack network.
```

Comment:
use either an IPv4 or a dual-stack network. 
AFAIK KubeVirt does not support dual-stack

Reply:
OpenShift Virtualization supports also dual-stack, see those FAQs: https://www.redhat.com/en/blog/red-hat-openshift-virtualization-faqs
and I think this is also true for the HyperShift use case.
@qinqon please keep me honest on this.